### DISCUSSION TITLE
Airflow Dagrun start automatically after unpause

### DISCUSSION THREAD

**Discussion by Dexter-07** [apache/airflow#41698] [Q&A]
My Dag schedule is 6PM daily EST time with catchup = False
I pause the dag for couple of days, when i un pause, airflow automatically starts the dagrun.
I believe that is because of the start and end data interval

I tried using LatestOnlyOperator but still no luck

Any idea how to avoid the situation ?
I want to make sure the dag should only run at the specified time when on unpause

**Comment by faze-akash** [comment]
@Dexter-07  Did you find any solution for this issue? I also want to avoid this automatic trigger when we unpause the DAG. I used the combination of DAG attributes like, depends_on_past=False, max_active_runs=1,catchup=false. These helps in avoiding the automatic trigger when we unpause but there's one catch. This combination fails if we are unpausing the DAG after few days let's say your DAG was paused for a week and then you unpause it a trigger would start. Another scenario is when you newly import a DAG and unpause it, a trigger is automatically triggered.So let me know if you find any solution
